OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -158,100 +1601,230 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <desc>Frame with uneven lines filled with a pattern of thin strings knotted by loops at its centre.
+                                </desc>
+                            </decoNote>
+                            
+                            <decoNote type="ornamentation" xml:id="d2">
+                                <locus target="#19va #29rb #40ra #49va #64ra #74rb"></locus>
+                                <desc>Smaller bands, marking incipits, present small variations on the pattern found on <locus target="#4r"></locus>.
+                                </desc>
+                            </decoNote>
+                            
+                        </decoDesc>
+                        
+                        <additions>
+                            <list>
+                                <item xml:id="e2">
+                                    <desc>
+                                        Omitted word(s) have been added by the hand that furnished the text with <foreign xml:lang="gez">mələkkət</foreign> 
+                                        in the margin (e.g. <locus target="#6r #7r #8r"/>) or interlineally (e.g. <locus target="#29ra #34rb #41ra"/>).
+                                        Different type of deletions: erasures (e.g. <locus target="#17vb #20rb #30va"/>), erasures with subsequent addition of line fillers 
+                                        (e.g. <locus target="#5va #16rb #24rb"/>),
+                                        expunctuation (<locus target="#29ra"/>), strike-through (<locus target="#32vb #66va"/>).
+                                        On <locus target="#45rb"/>, a rubricated word has been corrected by writing the correct form in black on top of the rubricated letters.
+                                    </desc>
+                                </item>
+                                <item xml:id="e3">
+                                    <desc>Pen trials by different hands with different inks (<locus from="2r" to="3v"/>).
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
                     </physDesc>
+                        
+                    <history>
+                        
+                    </history>
+                    </msPart>  
+                    
+                    <msPart xml:id="p2">
+                        <msIdentifier></msIdentifier>
+                        
+                        <msContents>
+                            <msItem xml:id="p2_i1">
+                            <locus from="94ra" to="95rb"/>
+                            <title type="complete" ref="LIT2814RepCh98"/>
+                            
+                                <msItem xml:id="p2_i1.1">
+                                <locus from="94ra" to="94va"/>
+                                <title type="complete"></title>
```

Comment:
there needs to be some title here (and in the items below): either a ref to an ID, or text written in the content of the element